eos/observable.cc

```diff
@@ -184,6 +185,86 @@ namespace eos
             make_observable("B->K::f_0(s)", "B->K",
                     &FormFactors<PToP>::f_0),
 
+            // B -> D^* Form Factors
+            make_observable("B->D^*::V(s)", "B->D^*",
+                            &FormFactors<PToV>::v),
+
+            make_observable("B->D^*::A_0(s)", "B->D^*",
+                            &FormFactors<PToV>::a_0),
+
+            make_observable("B->D^*::A_1(s)", "B->D^*",
+                            &FormFactors<PToV>::a_1),
+
+            make_observable("B->D^*::A_2(s)", "B->D^*",
+                            &FormFactors<PToV>::a_2),
+
+            make_observable("B->D^*::A_12(s)", "B->D^*",
+                            &FormFactors<PToV>::a_12),
+
+            make_observable("B->D^*::T_1(s)", "B->D^*",
+                            &FormFactors<PToV>::t_1),
+
+            make_observable("B->D^*::T_2(s)", "B->D^*",
+                            &FormFactors<PToV>::t_2),
+
+            make_observable("B->D^*::T_3(s)", "B->D^*",
+                            &FormFactors<PToV>::t_3),
+
+            make_observable("B->D^*::T_23(s)", "B->D^*",
+                            &FormFactors<PToV>::t_23),
+
+            make_observable("B->D^*::V(s)/A_1(s)", "B->D^*",
+                            &FormFactors<PToV>::v, &FormFactors<PToV>::a_1),
+
+            make_observable("B->D^*::A_2(s)/A_1(s)", "B->D^*",
+                            &FormFactors<PToV>::a_2, &FormFactors<PToV>::a_1),
+
+            make_observable("B->D^*::A_12(s)/A_1(s)", "B->D^*",
+                            &FormFactors<PToV>::a_12, &FormFactors<PToV>::a_1),
+
+            make_observable("B->D^*::T_23(s)/T_2(s)", "B->D^*",
+                            &FormFactors<PToV>::t_23, &FormFactors<PToV>::t_2),
+
+            // B -> rho Form Factors
```

Comment:
Again, please move B->rho below B->K^*, such that alphabetical order is maintained.